### PR TITLE
[issue 4074][pulsar-io]debezium document udpate

### PR DESCRIPTION
**Pull request by tuteng**
Fixes https://github.com/apache/pulsar/issues/4074

Master Issue: https://github.com/apache/pulsar/issues/4074

### Motivation

After move it out of Kafka-connect-source, debezium connector documentation need some update.
Update debezium mysql connector.

### Modifications

Move kafka-connect-adaptor to debezium-mysql

### CODE REVIEW

File: site2/docs/io-cdc-debezium.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Here is a configuration Json example:
         "pulsar.service.url": "pulsar://127.0.0.1:6650",
         "offset.storage.topic": "offset-topic"
     },
-    "archive": "connectors/pulsar-io-kafka-connect-adaptor-2.3.0-SNAPSHOT.nar"
+    "archive": "connectors/pulsar-io-debezium-mysql-2.4.0-SNAPSHOT.nar"
```
Comment (merlimat) [inline site2/docs/io-cdc-debezium.md]: We should replace `2.4.0-SNAPSHOT` with `{{pulsar:version}}`

Comment (merlimat) [reply site2/docs/io-cdc-debezium.md]: (that will be replace when the site is published)

**Comment by sijie** [comment]
@tuteng 

in the description of #4074 - "May need provide doc for debezium postgres connector, and update debezium mysql connector", we need add documentation for postgres connector. Can you please add the documentation for postgres cdc?

**Comment by tuteng** [comment]
> please add documentation for postgres.

I will update document of debezium on another branch. Some tests have not been completed yet.

**Comment by sijie** [comment]
run cpp tests
run java8 tests

**Comment by tuteng** [comment]
run java8 tests

**Pull request merged**